lisa/tools/modprobe.py

```diff
@@ -111,6 +111,31 @@ def load(
         )
         return True
 
+    def load_with_parameters(
```

Comment:
add the parameters as option parameter for load method

Reply:
`load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.

Reply:
You can throw an exception if parameters is specified when modules is a list.

Reply:
> `load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.

You can just join the mod list and parameters, and let modprobe return non-zero exit-code.

Reply:
> > `load` method can take list of modules as input. Parameters can be passed in `Modprobe` only with one module. Hence, a new method.
> 
> You can just join the mod list and parameters, and let modprobe return non-zero exit-code.

I dont think we should consciously append paremeters to mod list (`-a` option) and let modprobe fail.


> You can throw an exception if parameters is specified when modules is a list.

Sure, I will raise `UnsupportedOperationException`